interactive/plots/flow_diagram.py

```diff
@@ -1,798 +1,309 @@
+                        series=df[mapping[reaction["irr"]][0]],
+                        model_time_range_indices=model_time_range_indices,
+                        model_timestep=model_timestep,
+                    )
+                    if user_filter_toggle and not (
+                        average_flux > model_filter_range[0]
+                        and average_flux < model_filter_range[1]
+                    ):
+                        display_color = "#e0e0e0"
+                    width = (
+                        reaction["reactants"][reactant].get("yield", 1)
+                        * average_flux
+                    )
+                    displayWidth = scale_arrow_width(
+                        width=width,
+                        raw_flux_min_max=raw_flux_min_max,
+                        display_min_max=[1, user_width_scaling],
+                        user_arrow_scaling_type=user_arrow_scaling_type,
+                    )
+                    net.add_edge(
+                        reactant,
+                        reaction["label"],
+                        width=displayWidth,
+                        title="flux: " + str(width),
+                        color=display_color,
+                    )
+                # logger.info(f"average_flux: {average_flux}")
+            for product in reaction["products"]:
+                display_color = "#FF7F7F"
+                # reaction rate here is take as the last time entry from our
+                # simulation result.
+                # todo change flux calc for user selected time window
+                if "irr__" not in product:
+                    average_flux = get_flux(
+                        series=df[mapping[reaction["irr"]][0]],
+                        model_time_range_indices=model_time_range_indices,
+                        model_timestep=model_timestep,
+                    )
+                    if user_filter_toggle and not (
+                        average_flux > model_filter_range[0]
+                        and average_flux < model_filter_range[1]
+                    ):
+                        display_color = "#e0e0e0"
+                    width = (
+                        reaction["products"][product].get("yield", 1)
+                        * average_flux
+                    )
+                    displayWidth = scale_arrow_width(
+                        width=width,
+                        raw_flux_min_max=raw_flux_min_max,
+                        display_min_max=[1, user_width_scaling],
+                        user_arrow_scaling_type=user_arrow_scaling_type,
+                    )
+                    net.add_edge(
+                        reaction["label"],
+                        product,
+                        width=displayWidth,
+                        color=display_color,
+                        title="flux: " + str(width),
+                    )
+                # logger.info(f"average_flux: {average_flux}")
```

Comment:
Please delete commented out code. If it's debug information that could be useful, change the logging level to debug and uncomment